src/style.css

```diff
@@ -1,3 +1,94 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --blue: #00acdc;
+}
+
 body {
   margin: 0;
+  background-color: #e5e5e5;
+  font-family: Roboto, sans-serif, 'Times New Roman', Times, serif;
+}
+
+a,
```

Comment:
According to [CSS guidelines](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md#css), you should not use tag names for styling (except `html` and `body`). In your solution `a` and `ul` tags are used in CSS selectors - you should use classes instead.